src/test/java/com/google/api/codegen/testdata/library_gapic.yaml

```diff
@@ -504,6 +504,7 @@ interfaces:
       - go
       - java
       visibility: PACKAGE
+      deprecated: TRUE
```

Comment:
I would prefer a more flexible enumeration. Maybe:

`lifecycle: DEPRECATED`

That allows us to do something like this too:

`lifecycle: EXPERIMENTAL`

This also implies that the SurfaceNamer for each language needs to produce that language's string for each concept.